tests/integration/config/plugins/integration-tests/data/tests/tests_illegal_atrocity.txt

```diff
@@ -0,0 +1,386 @@
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 20
+				"thrust" 50
+				"turn" 1000
+				"energy generation" 10
+			outfits
+				Hyperdrive
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			position 0 0
+			engine -9 38 1
+			engine 9 38 1
+			system "Terra Incognita"
+			planet Ruin
+		account
+			credits 100000000
+			score 400
+			history
+
+		event "atrocity stuff"
+			government "Quarg"
+				atrocities
+					Hyperdrive
+				remove "enforces"
+		visited "Terra Incognita"
+		"visited planet" Ruin
+		"available mission" "%TEST%: Wardragon Scanning You!"
+			name "%TEST%: Wardragon Scanning You!"
+			uuid 9ba505d8-e565-4632-8c7a-c74072b85b5a
+			priority
+			to offer
+				"%TEST%: SCANNING ILLEGALS" != 0
+			destination Ruin
+			npc
+				uuid 01a921d2-b477-4d80-a19e-11aefb2a6464
+				succeed 1
+				government Quarg
+				personality derelict surveillance target
+				ship "Quarg Wardragon"
+					name Scanner
+					sprite ship/wardragon
+					thumbnail thumbnail/wardragon
+					uuid 7cc5c522-858d-419c-88a7-f240fc5799d7
+					attributes
+						category "Heavy Warship"
+						cost 5900000
+						mass 360
+						automaton 1
+						drag 9.3
+						"heat dissipation" 0.5
+						hull 50000
+						"outfit scan power" 1e+09
+						"outfit scan speed" 1e+09
+						"energy generation" 10
+						shields 160000
+					fuel 0
+					shields 0
+					hull 1000
+					position 0 0
+					explode "huge explosion" 1
+					system "Terra Incognita"
+			on offer
+				conversation
+					label 0
+					"You must be scanned or be killed."
+						accept
+		conditions
+			"%TEST%: SCANNING ILLEGALS"
+			"Ruin: Landing: offered"
+
+# just so the mission actually works
+mission "%TEST%: Wardragon Scanning You!"
+
+test "Land On Ruin helper"
+	status partial
+	description "Helper for landing on Ruin."
+	sequence
+		input
+			command land
+		label "landing"
+		# empty input to make time pass
+		input
+		branch "landing"
+			"flagship planet: Ruin" == 0
+
+test "Launch Quarg Scanning Mission And Board"
+	status partial
+	description "Starts the mission, accepts it and then boards the quarg ship to repair it."
+	sequence
+		# launch and accept the mission
+		input
+			key p
+		input
+			key d
+		call "Depart"
+		# board the ship so we're sure we get close to it to get scanned
+		input
+			command board
+		# Wait for 5 seconds so it should be repaired and have scanned us
+		# is there a way to know when a dialog pops up?
+		apply
+			"test: steps to wait" = 10 * 30
+		label "floating"
+		# empty input to make time pass
+		input
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch "floating"
+			"test: steps to wait" > 0
+		input
+			key "return"
+
+test "Boarding Test - Repair Friendly Ship"
+	status active
+	description "Test if you can repair a disabled Quarg ship trough an assist derelict mission."
+	sequence
+		inject "quarg rescue"
+		call "Load First Savegame"
+		watchdog 6000
+		call "Launch Quarg Scanning Mission And Board"
+		apply
```

Comment:
Could use a comment on that the test-data is copied into the temporary conditions for debugging when an issue happens. (The test-framework might in the future also print all conditions that were used, and someone looking at this code in the future might then not understand why this is done here.)

Reply:
Took me a bit to understand why it was done in the sell ship integration tests as well indeed